### ISSUE TITLE
lightgbm params

### ISSUE THREAD

**Issue by inoddy** [ModelOriented/forester#95]
library(forester)
data("titanic", package = 'DALEX') 
train_output <- train(data = titanic, "survived", type = "classification", engine = c("ranger", "xgboost", "decision_tree", "lightgbm", "catboost"))

produced after a few minutes

Error in lightgbm::lgb.train(params = params, data = data$lightgbm_data,  : 
  object 'params' not found

yet if I omit the type argument it runs ok and does a classification

**Comment by HubertR21** [comment]
There was a mistake in the documentation. Instead of 'classification' there should be 'binary_clf' (as there are plans for adding multi-label classification in the future). The fix of that issue will be available in the next package version.
